.storybook/main.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 module.exports = {
   stories: [
     '../{packages,packages-node}/!(ajax)*/README.md',
-    '../{packages,packages-node}/*/docs/*.md',
+    '../{packages,packages-node}/!(ajax)*/docs/*.md',
```

Comment:
fixed https://github.com/ing-bank/lion/pull/1228 so maybe leave this out otherwise we get merge conflict